typescript/sdk/src/types.ts

```diff
@@ -1,12 +1,12 @@
 import type { ethers } from 'ethers';
 
-import type { ChainId, Domain } from '@hyperlane-xyz/utils';
+import type { Domain } from '@hyperlane-xyz/utils';
 
 // An alias for string to clarify type is a chain name
 export type ChainName = string;
 // A map of chain names to a value type
-export type ChainMap<Value> = Record<string, Value>;
+export type ChainMap<Value> = Record<ChainName, Value>;
 
-export type ChainNameOrId = ChainName | ChainId | Domain;
+export type ChainNameOrDomain = ChainName | Domain;
```

Comment:
I personally think changing this types name is not really necessary and is the main reason this diff is so large